Application/VixenApplication/VixenApplication.cs

```diff
@@ -140,7 +141,10 @@ public VixenApplication()
 				form.Dispose(); 
 				Environment.Exit(0);
 			}
-			
+
+			// Give the fixture specification manager the root profile data directory
+			FixtureSpecificationManager.Instance().InitializeProfilePath(_rootDataDirectory);
+

```

Comment:
I updated the manager to use DataRootPath.  There is a project dependency between VixenApplication and the FixtureSpecificationManager because all of the ElementTemplates live inside VixenApplication.  We may want to make those modules in the future.